docs/development/release-process.rst

```diff
@@ -64,26 +64,24 @@ Release Process
 Creating a new release
 ----------------------
 
-#. On the current pip ``master`` branch, generate a new ``AUTHORS.txt`` by
-   running ``invoke generate.authors`` and commit the results.
-#. On the current pip ``master`` branch, make a new commit which bumps the
-   version in ``pip/__init__.py`` to the release version and adjust the
-   ``CHANGES.txt`` file to reflect the current date. The release version should
-   follow a YY.N scheme, where YY is the two digit year, and N is the Nth
-   release within that year.
-#. On the current pip ``master`` branch, generate a new ``NEWS.rst`` by running
-   ``invoke generate.news`` and commit the results.
-#. Create a signed tag of the ``master`` branch of the form ``YY.N`` using the
-   command ``git tag -s YY.N``.
-#. Checkout the tag using ``git checkout YY.N`` and create the distribution
-   files using ``python setup.py sdist bdist_wheel``.
+#. Checkout the current pip ``master`` branch.
```

Comment:
It might be worth mentioning what tools you need installed. I worked in a clean virtualenv when doing the release and I did have to install tools as I went along. I'm not 100% sure I remember what I needed, but towncrier and invoke were two I know I needed.

Reply:
Indeed. Will do.